package/src/components/Message/MessageSimple/MessageContent.tsx

```diff
@@ -297,17 +310,11 @@ const MessageContentWithContext = <
             styles.containerInner,
             {
               backgroundColor,
-              borderBottomLeftRadius:
-                (groupStyle === 'left_bottom' || groupStyle === 'left_single') &&
-                (!hasThreadReplies || threadList)
-                  ? borderRadiusS
-                  : borderRadiusL,
-              borderBottomRightRadius:
-                (groupStyle === 'right_bottom' || groupStyle === 'right_single') &&
-                (!hasThreadReplies || threadList)
-                  ? borderRadiusS
-                  : borderRadiusL,
-              borderColor: isMyMessage && !error ? backgroundColor : grey_whisper,
+              borderBottomLeftRadius: isBorderBottomLeftRadius ? borderRadiusS : borderRadiusL,
+              borderBottomRightRadius: isBorderBottomRightRadius ? borderRadiusS : borderRadiusL,
+              borderColor: isBorderColor ? backgroundColor : grey_whisper,
+              borderTopLeftRadius: isBorderTopLeftRadius ? borderRadiusS : borderRadiusL,
+              borderTopRightRadius: isBorderTopRightRadius ? borderRadiusS : borderRadiusL,
```

Comment:
Discussion: let's also give the option to change everything at the same time via `borderRadius` ?

Reply:
This makes sense